### PR TITLE
fix luacheck warnings : fix comments accordingly

### PR DESCRIPTION
**Pull request by rubo77**
Due to the fixes in 4249d65af73ee9228d827af913b7caf2aac00809 the comments have to change too

### CODE REVIEW

File: package/gluon-web/luasrc/usr/lib/lua/gluon/web/http/protocol.lua [hunk 1]
```diff
@@ -223,9 +223,7 @@ local function mimedecode_message_body(src, msg, filecb)
 			until not spos
 
 			if found then
-				-- We found at least some boundary. Save
-				-- the unparsed remaining data for the
-				-- next chunk.
+				-- We found at least some boundary.
```
Comment (NeoRaider) [inline package/gluon-web/luasrc/usr/lib/lua/gluon/web/http/protocol.lua]: The comment is still correct, there is nothing to change here.

Comment (rubo77) [reply package/gluon-web/luasrc/usr/lib/lua/gluon/web/http/protocol.lua]: But where is the remaining data used after that line? the if clause ends and else is another case.

Comment (NeoRaider) [reply package/gluon-web/luasrc/usr/lib/lua/gluon/web/http/protocol.lua]: lchunk is used again in the next call of snk. Unlike data, lchunk is an upvalue.

Comment (rubo77) [reply package/gluon-web/luasrc/usr/lib/lua/gluon/web/http/protocol.lua]: AH, ow I get it: the comment was misleading, I thought, with the remaining "data" they mean the variable "data", but the comment is not about the variable but just uses the word "data"

maybe we should rephrase this then?

> save the unparsed remaining data-part for the next chunk.

Comment (rubo77) [reply package/gluon-web/luasrc/usr/lib/lua/gluon/web/http/protocol.lua]: changed to "the remaining string"

Comment (NeoRaider) [reply package/gluon-web/luasrc/usr/lib/lua/gluon/web/http/protocol.lua]: Even if it refered to the "variable" data, the comment would still be accurate: The remaining (variable) "data" is saved (by assigning it to a different variable). Please just leave this comment unchanged.

Comment (rubo77) [reply package/gluon-web/luasrc/usr/lib/lua/gluon/web/http/protocol.lua]: ok, done ;)

**Pull request merged**